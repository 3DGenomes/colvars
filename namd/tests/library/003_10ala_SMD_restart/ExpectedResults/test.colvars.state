configuration {
  step           10
  dt 5.000000e-01
}

colvar {
  name rmsd
  x  3.82640484672394e-02
}

harmonic {
  configuration {
    name harmonic1
    centers  1.00000000000000e-01
    centers_raw  1.00000000000000e-01
    accumulatedWork 3.54824797528161e+01
  }
}

