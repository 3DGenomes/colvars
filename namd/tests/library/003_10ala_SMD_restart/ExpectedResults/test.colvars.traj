# step         rmsd                  fs_rmsd               fa_rmsd                E_harmonic1           x0_rmsd               W_harmonic1          
           0    1.58959746069124e-16  0.00000000000000e+00 -1.58959746069124e-12  1.26341004351803e-28  0.00000000000000e+00 -1.58959746069124e-14
           1    3.16367462765507e-03  4.31591398173263e+01  6.83632537234493e+01  2.33676722982835e-01  1.00000000000000e-02 6.83632537234477e-01
           2    6.43055388319465e-03  2.19865413831198e+01  1.35694461168054e+02  9.20649339584420e-01  2.00000000000000e-02 2.04057714891501e+00
           3    9.79614980051385e-03 -4.30058306046606e+00  2.02038501994862e+02  2.04097781441638e+00  3.00000000000000e-02 4.06096216886363e+00
           4    1.32400619260051e-02 -3.37493385624498e+01  2.67599380739949e+02  3.58047142862021e+00  4.00000000000000e-02 6.73695597626312e+00
           5    1.67527472991108e-02 -6.22967338815067e+01  3.32472527008892e+02  5.52689906078392e+00  5.00000000000000e-02 1.00616812463520e+01
           6    2.05399092691416e-02 -8.88512605975345e+01  3.94600907308584e+02  7.78549380243787e+00  6.00000000000000e-02 1.40076903194379e+01
           7    2.46473595457114e-02 -1.17398919239643e+02  4.53526404542887e+02  1.02843099808799e+01  7.00000000000000e-02 1.85429543648667e+01
           8    2.88887062712024e-02 -1.50035359611463e+02  5.11112937287976e+02  1.30618217331571e+01  8.00000000000000e-02 2.36540837377465e+01
           9    3.34519913820647e-02 -1.84889273991458e+02  5.65480086179353e+02  1.59883863932704e+01  9.00000000000000e-02 2.93088845995400e+01
          10    3.82640484672394e-02 -2.23258039694226e+02  6.17359515327606e+02  1.90566385582768e+01  1.00000000000000e-01 3.54824797528161e+01
