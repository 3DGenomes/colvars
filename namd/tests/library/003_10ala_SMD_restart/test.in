colvarsTrajFrequency    1
#colvarsRestartFrequency 10


colvar {
    name rmsd

    outputSystemForce on
    outputAppliedForce on
    width 0.01

    lowerBoundary 0.0
    upperBoundary 0.1

    hardLowerBoundary yes
    expandBoundaries yes

#    lowerWallConstant 10.0
    upperWallConstant 10.0

    rmsd {
        atoms {
            enablefitGradients yes
            psfSegID BH
            atomNameResidueRange CA 1-10
        }
        refpositionsfile ../Common/pi-ideal.pdb
    }
} 

harmonic {
    colvars        rmsd
    centers        0.0
    targetCenters  0.2
    targetNumSteps 20
    outputEnergy   yes
    outputAccumulatedWork  yes
    outputCenters  yes
}
