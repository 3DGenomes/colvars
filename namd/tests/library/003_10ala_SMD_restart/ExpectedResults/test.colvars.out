colvars: ----------------------------------------------------------------------
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 40000 [default]
colvars: # colvarsTrajAppend = off [default]
colvars: The restart output state file will be "test.colvars.state".
colvars: The final output state file will be "test.colvars.state".
colvars: The trajectory file will be "test.colvars.traj".
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = rmsd
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # disableForces = off [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on
colvars:       Atom group "atoms" defined, 10 atoms initialized: total mass = 120.11.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = ../Common/da.pdb
colvars:     # refPositionsCol = "" [default]
colvars:     Enabling "centerReference" and "rotateReference", to minimize RMSD before calculating it as a variable: if this is not the desired behavior, disable them explicitly within the "atoms" block.
colvars:   All components initialized.
colvars:   # width = 0.01
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 0 [default]
colvars:   # upperBoundary = 0.1
colvars:   # upperWallConstant = 10
colvars:   # upperWall = 0.1 [default]
colvars:   # hardLowerBoundary = on
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = on
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = on
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { rmsd }
colvars:   # outputEnergy = on
colvars:   # forceConstant = 1 [default]
colvars:   The force constant for colvar "rmsd" will be rescaled to 10000 according to the specified width.
colvars:   # centers = { 0 }
colvars:   # targetCenters = { 0.2 }
colvars:   # targetForceConstant = 0 [default]
colvars:   # targetNumSteps = 20
colvars:   # targetNumStages = 0 [default]
colvars:   # targetForceExponent = 1 [default]
colvars:   # outputCenters = on
colvars:   # outputAccumulatedWork = on
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module initialized.
colvars: ----------------------------------------------------------------------
colvars: Saving collective variables state to "test.colvars.state".
